api/lanes.py

```diff
@@ -890,10 +890,11 @@ def fetch_recommendations(self, _db):
     def apply_filters(self, _db, qu, facets, pagination, featured=False):
         if not self.recommendations:
             return None
-
+        from core.model import MaterializedWorkWithGenre as wm
         qu = qu.join(LicensePool.identifier)
         qu = Work.from_identifiers(
-            _db, self.recommendations, base_query=qu
+            _db, self.recommendations, base_query=qu,
+            identifier_id_field=mw.identifier_id
```

Comment:
The variable name is off here. It should be `wm`. Or the one above should be `mw`.